doc/source/ray-overview/getting-started.md

```diff
@@ -5,93 +5,14 @@
 
 # Getting Started
 Use Ray to scale applications on your laptop or the cloud. Choose the right guide for your task.
-* Scale end-to-end ML applications: [Ray AI Runtime Quickstart](#ray-ai-runtime-quickstart)
-* Scale single ML workloads: [Ray Libraries Quickstart](#ray-libraries-quickstart)
+* Scale ML workloads: [Ray Libraries Quickstart](#ray-libraries-quickstart)
 * Scale general Python applications: [Ray Core Quickstart](#ray-core-quickstart)
 * Deploy to the cloud: [Ray Clusters Quickstart](#ray-cluster-quickstart)
 * Debug and monitor applications: [Debugging and Monitoring Quickstart](#debugging-and-monitoring-quickstart)
-## Ray AI Runtime Quickstart
-
-Explore Ray's full suite of libraries for end-to-end ML pipelines, with the `air` packages:
-
-```
-pip install -U "ray[air]"
-```
-
-`````{dropdown} Efficiently process your data into features.
```

Comment:
This is somewhat redundant with the following section, so I removed it. That being said, we still need to have some end-to-end examples for using Ray for complete ML workflows.